oneandone/resource_oneandone_server.go

```diff
@@ -526,6 +589,33 @@ func readIps(ips []oneandone.ServerIp) []map[string]interface{} {
 	return raw
 }
 
+func fixedsize2Id(config *Config, fixedsize string) (string, error) {
+	fixedsizes, err := config.API.ListFixedInstanceSizes()
+	if err != nil {
+		return "", fmt.Errorf("Could not fetch FixedInstanceSizes: %s ", err)
+	}
+	fixedsize = strings.ToLower(fixedsize)
+	for _, size := range fixedsizes {
+		if strings.ToLower(size.Name) == fixedsize {
+			return size.Id, nil
+		}
+	}
+	return "", fmt.Errorf("FixedInstanceName not found: %s ", fixedsize)
+}
```

Comment:
You could base it off of [this](https://github.com/terraform-providers/terraform-provider-aws/blob/master/aws/data_source_aws_acm_certificate.go)